plugins/BEdita/API/src/Controller/StatusController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2016 ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+namespace BEdita\API\Controller;
+
+use BEdita\Core\Utility\System;
+
+/**
+ * Controller for `/status` endpoint.
+ *
+ * @since 4.0.0
+ */
+class StatusController extends AppController
+{
+
+    /**
+     * Show system status info
+     *
+     * @return void
+     */
+    public function index()
+    {
+        $status = System::status();
```

Comment:
Missing check on request method `$this->request->allowMethod('get');`

For example `POST /status HTTP` responds as GET request.
Instead it should respond as `405 Method not Allowed`.